# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/base/reiserfsprogs/no-coredump.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- reiserfsprogs-3.6.11/lib/misc.c.orig	2003-08-16 11:58:25.000000000 +0200
+++ reiserfsprogs-3.6.11/lib/misc.c	2003-08-16 11:59:32.000000000 +0200
@@ -50,7 +50,7 @@
     va_end (args);
 
     fprintf (stderr, "\n%s\n", buf);
-    abort ();
+    exit (-1);
 }
 
 
